### PR TITLE
Remove unused sched module

### PR DESCRIPTION
**Pull request by jepebe**
The sched module in libres had multiple classes that were not used by the rest of library. This PR removes those.

### CODE REVIEW

File: python/res/sched/CMakeLists.txt [hunk 1]
```diff
@@ -2,8 +2,7 @@ set(PYTHON_SOURCES
     __init__.py
     history.py
     history_source_enum.py
-    sched_file.py
-)
+        )
```
Comment (markusdregi) [inline python/res/sched/CMakeLists.txt]: Some odd indentation 🤷

**Review by markusdregi — COMMENTED** [review]
This looks good 👍 Found an odd indentation; when that is fixed this one is good to go!

File: python/res/sched/CMakeLists.txt (hunk 1 above)
Comment (jepebe) [reply python/res/sched/CMakeLists.txt]: Yes, that was odd.

**Review by markusdregi — APPROVED** [review]
🚢

**Pull request merged**